js/app.js

```diff
@@ -1,58 +1,62 @@
-function Project(project){
-  for(key in project){
-    this[key] = project[key];
+'use strict';
+(function(module){
+  function Project(project){
+    for(var key in project){
+      this[key] = project[key];
+    }
   }
 
-}
+  Project.all = [];
 
-Project.all = [];
+  Project.prototype.toHtml = function(){
+    var $source = $('#project-template').html();
+    var template = Handlebars.compile($source);
+    return template(this);
+  };
 
-Project.prototype.toHtml = function(){
-  var $source = $('#project-template').html();
-  var template = Handlebars.compile($source);
-  return template(this);
-};
+  Project.loadAll = function(data){
+    data.forEach(function(ele) {
```

Comment:
You're trying to turn an array of JS objects into an array of Projects. In other words, you're transforming one collection to another. What array method can we use in that case?